tests/schemas/valid/test_dynamic_answers_list_source.json

```diff
@@ -0,0 +1,322 @@
+                  ]
+                }
+              },
+              "remove_block": {
+                "id": "remove-supermarket",
+                "type": "ListRemoveQuestion",
+                "cancel_text": "Don’t need to remove this supermarket?",
+                "question": {
+                  "id": "remove-question",
+                  "type": "General",
+                  "title": "Are you sure you want to remove this supermarket?",
+                  "warning": "All of the information about this supermarket will be deleted",
+                  "answers": [
+                    {
+                      "id": "remove-confirmation",
+                      "mandatory": true,
+                      "type": "Radio",
+                      "options": [
+                        {
+                          "label": "Yes",
+                          "value": "Yes",
+                          "action": {
+                            "type": "RemoveListItemAndAnswers"
+                          }
+                        },
+                        {
+                          "label": "No",
+                          "value": "No"
+                        }
+                      ]
+                    }
+                  ]
+                }
+              },
+              "summary": {
+                "title": "Supermarkets",
+                "item_title": {
+                  "text": "{supermarket_name}",
+                  "placeholders": [
+                    {
+                      "placeholder": "supermarket_name",
+                      "transforms": [
+                        {
+                          "arguments": {
+                            "delimiter": " ",
+                            "list_to_concatenate": [
+                              {
+                                "source": "answers",
+                                "identifier": "supermarket-name"
+                              }
+                            ]
+                          },
+                          "transform": "concatenate_list"
+                        }
+                      ]
+                    }
+                  ]
+                }
+              }
+            },
+            {
+              "type": "Question",
+              "id": "dynamic-answer",
+              "skip_conditions": {
+                "when": {
+                  "==": [
+                    {
+                      "count": [
+                        {
+                          "source": "list",
+                          "identifier": "supermarkets"
+                        }
+                      ]
+                    },
+                    0
+                  ]
+                }
+              },
+              "question": {
+                "dynamic_answers": {
+                  "values": {
+                    "source": "list",
+                    "identifier": "supermarkets"
+                  },
+                  "answers": [
```

Comment:
We don't have an example where dynamic answers have more than 1 answer. We should add another answer of a different type, i.e Numbers. Same for runner.

Reply:
Added, thanks.